dizoo/evogym/envs/evogym_env.py

```diff
@@ -0,0 +1,174 @@
+from typing import Any, Union, List, Optional
+import copy
+import numpy as np
+from easydict import EasyDict
+import gym
+import evogym.envs
+from evogym import WorldObject, sample_robot
+from .viewer import DingEvoViewer
+from evogym.sim import EvoSim
+import os
+from ding.envs import BaseEnv, BaseEnvTimestep, FinalEvalRewardEnv
+from ding.envs.common.common_function import affine_transform
+from ding.torch_utils import to_ndarray, to_list
+from ding.utils import ENV_REGISTRY
+
+
+@ENV_REGISTRY.register('evogym')
+class EvoGymEnv(BaseEnv):
+
+    @classmethod
+    def default_config(cls: type) -> EasyDict:
```

Comment:
Where is this classmethod being used?

Reply:
I saw that also in other envs it is not used, isn't according to the user intentions whether to use this class or not?